taxi/models.py

```diff
@@ -0,0 +1,28 @@
+from django.conf import settings
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    country = models.CharField(max_length=255)
+
+    def __str__(self) -> str:
+        return self.name
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(max_length=255, unique=True, null=True)

```

Comment:
You've extended the `AbstractUser` model to create your `Driver` model, which is a great approach. However, you might want to consider renaming the `Driver` model to `User` or `CustomUser` to avoid confusion. This is because in Django, the user model is often used to handle authentication and authorization, and not just to represent the concept of a 'driver' in your application. This is more of a code design issue.